.github/workflows/nightly_tests.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           python -m uv pip install -e [quality,test]
           pip uninstall accelerate -y && python -m uv pip install -U accelerate@git+https://github.com/huggingface/accelerate.git
           python -m uv pip install pytest-reportlog
+          python -m uv pip install phonemizer
```

Comment:
We can update the test dep here
https://github.com/huggingface/diffusers/blob/edb8c1bce67e81f0de90a7e4c16b2f6537d39f2d/setup.py#L212

And it will get picked up when running `python -m uv pip install -e [quality,test]`

Reply:
Nice, thanks!